maker/src/routes.rs

```diff
@@ -52,6 +56,7 @@ pub fn router(
         .route("/api/pay-invoice/:invoice", post(pay_invoice))
         .route("/api/sync-on-chain", post(sync_on_chain))
         .route("/metrics", get(get_metrics))
+        .route("/health", get(get_health))
```

Comment:
For whom is this new endpoint?

Reply:
it's useful in development, testing, and manual checking if we want to quickly see the state.